scanpy/preprocessing/docs.py

```diff
@@ -0,0 +1,88 @@
+doc_norm_bulk = """\
+Normalize total counts per cell.
+
+Normalize each cell by total counts over genes, so that every cell has
+the same total count after normalization.
+
+Similar functions are used, for example, by Seurat [Satija15]_, Cell Ranger
+[Zheng17]_ or SPRING [Weinreb17]_.
+
+Parameters
+----------
+data : :class:`~anndata.AnnData`
+    The annotated data matrix of shape `n_obs` × `n_vars`. Rows correspond
+    to cells and columns to genes.
+cell_sum_after : `float` or `None`, optional (default: `None`)
+    If `None`, after normalization, each cell has a total count equal
+    to the median of the *counts_per_cell* before normalization.
+min_counts : `int`, optional (default: 1)
+    Cells with counts less than `min_counts` are filtered out during
+    normalization.
+key_n_counts : `str`, optional (default: `None`)
+    Name of the field in `adata.obs` where the total counts per cell are
+    stored.
+inplace : `bool`, optional (default: `True`)
```

Comment:
`inplace` should appear at the very end of the list. It's a generic argument like `copy` and will appear in almost every function.